package/Dockerfile

```diff
@@ -151,8 +151,15 @@ RUN curl -sLf ${!HELM_URL_V2} > /usr/bin/rancher-helm && \
     chmod +x /usr/bin/rancher-helm /usr/bin/rancher-tiller
 
 # set up helm 3
-RUN curl ${HELM_URL_V3} | tar xvzf - --strip-components=1 -C /usr/bin && \
```

Comment:
Since you removed this the `HELM_URL_V3` env var is no longer referenced, it would be good to also remove that (line 129) just so it doesn't cause any confusion.